buf.gen.yaml

```diff
@@ -9,3 +9,12 @@ plugins:
   - name: grpc-gateway
     out: code/go/0chain.net/miner/proto/api/src/proto
     opt: paths=source_relative,allow_delete_body=true
+  # - name: go
```

Comment:
The Sharder code seems to be generated even without specifing in `gen` file :confused: . A bit confused here.

Reply:
I think this can be solved if we created a shared package for miner, sharder .. proto  if we have a directory like:
`code/go/0chain.net/grpc/` inside it `miner/proto/v1` & `sharder/proto/v1` and Go code will be generated in `code/go/0chain.net/grpc/{miner|sharder}/src`

I think will need to change `buf.yaml` and create new one inside `proto/{miner|sharder}`